hack/test/e2e-qemu.sh

```diff
@@ -35,6 +35,16 @@ case "${WITH_UEFI:-false}" in
     ;;
 esac
 
+case "${USE_DISK_IMAGE:-false}" in
+  false)
+    DISK_IMAGE_FLAG=
+    ;;
+  *)
+    tar -xf _out/metal-amd64.tar.gz -C _out/ --strip-components=1
```

Comment:
I guess that's most likely a wrong assumption that this image exists at this point.
Can I call `make image-metal` here? Or that's a bad idea?

Reply:
it should be created as part of the build by `images-amd64` step, so we could rely on it I think